### ISSUE TITLE
程序运行内存占用过多的问题

### ISSUE THREAD

**Issue by RenDav** [CanCanZeng/ACMP_colmap#2]
您好，请问您使用ACMP算法计算深度图时会出现内存不够用的情况么，我选择了81张4864*3648的影像，64G内存不够用

**Comment by CanCanZeng** [comment]
Yes, this program saves all raw data and intermediate data in memory, so it may crash for very large dataset.

**Comment by CanCanZeng** [comment]
> Yes, this program saves all raw data and intermediate data in memory, so it may crash for very large dataset.

Sorry the answer is not right, it may crash in depth fusion stage, but not for depth estimation. I will check the code again.

**Comment by CanCanZeng** [comment]
Hi, I checked the code, there is no memroy leak or other thing. The algorithm do need so much memory, so I cannot help. But I pushed a new commit to reduce the memory usage silightly.
